src/settings.py

```diff
@@ -236,24 +236,55 @@
 DISABLED_ROLES = []
 
 NO_VICTIMS_MESSAGES = ("The body of a young penguin pet is found.",
+                       "Paw prints are found circling around the village.",
                        "A pool of blood and wolf paw prints are found.",
+                       "The body of a slain cat is found.",
+                       "Some house doors have been opened, but nothing has changed.",
+                       "A scent much like that of a wolf permeates the air.",
+                       "Half-buried wolf droppings are found.",
                        "Traces of wolf fur are found.")
 LYNCH_MESSAGES = ("The villagers, after much debate, finally decide on lynching \u0002{0}\u0002, who turned out to be... a{1} \u0002{2}\u0002.",
+                  "After a prolonged struggle, \u0002{0}\u0002 is forced to the gallows, and is discovered after death to be a{1} \u0002{2}\u0002.",
+                  "The villagers have chosen to hang \u0002{0}\u0002, but the rope stretches and breaks. However, the ensuing fall kills the \u0002{2}\u0002.",
```

Comment:
I suggest rewriting this as something along the lines of "the villagers choose to hang so-and-so; however, the rope stretches and breaks, and the ensuing fall kills the such-and-such for them." Same for the noreveal.